src/bundler.ts

```diff
@@ -70,7 +79,13 @@ export class Bundler {
         }
     }
 
-    private async bundle(filePath: string, content: string, dedupeFiles: string[], includePaths: string[]): Promise<BundleResult> {
+    private async bundle(
+        filePath: string,
+        content: string,
+        dedupeFiles: string[],
+        includePaths: string[],
+        ignoredImports: RegExp[]
+    ): Promise<BundleResult> {

```

Comment:
Maybe we should update arguments to an `options` variable? 
That will be a breaking change.

/cc @DovydasNavickas

Reply:
This is probably a really good idea. Personally it's not something I'd want to cover in this PR.